sandboxes/scripts/creators/industrial/create_industrial_venues.py

```diff
@@ -1,14 +1,30 @@
 import re
 
 from models.pc_object import PcObject
+from models import Offerer
 from sandboxes.scripts.mocks.venue_mocks import MOCK_NAMES
 from utils.logger import logger
 from tests.test_utils import create_venue, create_bank_information
 
 OFFERERS_WITH_PHYSICAL_VENUE_REMOVE_MODULO = 3
 OFFERERS_WITH_PHYSICAL_VENUE_WITH_SIRET_REMOVE_MODULO = OFFERERS_WITH_PHYSICAL_VENUE_REMOVE_MODULO * 2
 
-def create_industrial_venues(offerers_by_name):
+
+def make_venue_virtual(offerer: Offerer, venue_by_name: dict, venue_name: str) -> dict:
+    venue_by_name["{} (Offre en ligne)".format(venue_name)] = create_venue(
+        offerer,
+        address=None,
+        city=None,
+        is_virtual=True,
+        latitude=None,
+        longitude=None,
+        postal_code=None,
+        name="{} (Offre en ligne)".format(venue_name),
+        siret=None
+    )
```

Comment:
```suggestion
    def make_virtual_venue(offerer: Offerer, venue_by_name: dict, venue_name: str) -> dict:
    name = "{} (Offre en ligne)".format(venue_name)
    venue_by_name[name] = create_venue(
        offerer,
        name=name,
        address=None,
        postal_code=None,
        city=None,
        departement_code=None,
        is_virtual=True,
        longitude=None,
        latitude=None,
        siret=None
    )
```
J'ai renommé le nom de la fonction, extrait le `name` pour factoriser et j'ai rangé les paramètres dans le même ordre que la signature.

Reply:
Pour challenger encore plus, ta signature de retour n'est pas bonne car elle ne retourne pas un dict.
J'aurai préféré que ta fonction retourne `create_venue()` et qu'à la ligne 93, tu l'affectes à `venue_by_name["{} (Offre en ligne)".format(venue_name)]`, ce qui est beaucoup simple à tester.

Reply:
Et au final (désolé, je te mets tout mon cheminement) mais je ne vois pas l'intérêt de l'avoir extractée en fonction ?

Reply:
Bien vu pour le name, c'est plus lisible je trouve

Reply:
>  je ne vois pas l'intérêt de l'avoir extractée en fonction ?

parceque create_industrial_venues faisait déjà presque 70 lignes et pour mettre en évidence qu'on créé deux types de lieux